packages/compass-collection/src/stores/collection-tab.ts

```diff
@@ -9,6 +9,14 @@ import reducer, {
 import type { Collection } from '@mongodb-js/compass-app-stores/provider';
 import type { ActivateHelpers } from '@mongodb-js/compass-app-registry';
 import type { workspacesServiceLocator } from '@mongodb-js/compass-workspaces/provider';
+import type { experimentationServiceLocator } from '@mongodb-js/compass-telemetry';
+import type { connectionInfoRefLocator } from '@mongodb-js/compass-connections/provider';
+import type { Logger } from '@mongodb-js/compass-logging/provider';
+import {
+  isAIFeatureEnabled,
+  type PreferencesAccess,
+} from 'compass-preferences-model/provider';
+import { TestName } from '@mongodb-js/compass-telemetry/provider';
```

Comment:
TestName is a very generic name for this type, can we add experimentation to it to make it clearer what this refers to?